src/CalcViewModel/DateCalculatorViewModel.cpp

```diff
@@ -56,19 +52,17 @@ DateCalculatorViewModel::DateCalculatorViewModel() :
 
     // Initialize Date Calc engine
     m_dateCalcEngine = make_shared<DateCalculationEngine>(localizationSettings.GetCalendarIdentifier());
-
     // Initialize dates of DatePicker controls to today's date
     auto calendar = ref new Calendar();
+    calendar->ChangeTimeZone("UTC");
```

Comment:
The comment at line 83 (that you referenced in another comment) should be removed now.  Leaving it as is will be confusing, because you now _are_ changing he timezone to `UTC` before calling `ClipTime`.  Can you just add a simpler comment here that it's being done to avoid issues around Daylight Savings Time?